### ISSUE TITLE
Charm incompatible with VS2/Eureka

### ISSUE THREAD

**Issue by badgermemelord** [ValkyrienSkies/Valkyrien-Skies-2#354]
Crash on startup when the mod is present.
Log:
[latest.log](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/files/10420602/latest.log)
Crash report:
[crash-2023-01-15_17.09.24-client.txt](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/files/10420604/crash-2023-01-15_17.09.24-client.txt)

**Comment by error755** [comment]
it also happens to me i saw a post saying that i needed another launcher but i dont think it works but maybe you can get a client that actually works try it ig

**Comment by playernik2003** [comment]
Did you find a solution yet im facing the same problem

**Comment by error755** [comment]
Yes in the curseforge page of the mod you can find a red text saying that you need kotlin forge to run this mod install it and you can use the mod

**Comment by playernik2003** [comment]
I will try it out later but i am playing on fabric so i think i could fabric kotlin and fabric kotlin language will try it later

**Comment by playernik2003** [comment]
@error755 i tried it but its still crashing for me

**Comment by error755** [comment]
Then  i cant do anything more to help you im sorry

**Comment by playernik2003** [comment]
No problem XD
Thanks for the attempt to help.me tho XD

**Comment by AbyssalLight** [comment]
Is this bug still being worked on? Cause I have the same problem, even tried using kotlin for forge and it still failed. I thought this would be fixed fast due to the popularity of charm on fabric

**Comment by Acuadragon100** [comment]
The issue is caused by Valkyrien Skies shadowing an outdated version of Google Reflect, which it really shouldn't do...

Deleting the com/google directory from the compiled Valkyrien Skies jar file is a decent workaround for now.
